README.md

```diff
@@ -1,3 +1,3 @@
-# netuitive.packages.windows
+# CloudWisdom.packages.windows
```

Comment:
Keep this as netuitive, it's part of a fully qualified name.

Reply:
changed back to netuitive